src/components/select/select.spec.ts

```diff
@@ -476,6 +542,24 @@ describe('MdlSelect', () => {
 @Component({
     selector: 'test-single-component',
     template: `
+        <mdl-select label="{{label}}" floating-label [autoselect]="false" [(ngModel)]="selectedValue">
```

Comment:
don't you think it would be better to set autoselect to true if you want this feature and set it to false or leave the attribute autoselect away if you don't want the autoselect feature? For me this would be a more natural way how to use additional features...